LDAR_Sim/src/programs/method.py

```diff
@@ -277,7 +277,7 @@ def deploy_crews(self, workplan: Workplan, weather, daylight) -> CrewDeploymentS
             # indicate that the particular site needs to be requeued with higher priority
             incompleteSurveys.append((survey_report, survey_plan))
             if survey_report.survey_complete:
-                self._site_survey_reports.append(survey_report)
+                survey_report.follow_up_method = self.get_follow_up_method_name()
```

Comment:
This always returns "N/A". I don't believe it's being used anywhere.

Reply:
You are correct - Will remove